lib/state/index.ts

```diff
@@ -68,7 +67,8 @@ export const store = createStore(
         [path]: omit(state[path], 'focusModeEnabled'),
       }),
     }),
-    applyMiddleware(thunk, uiMiddleware)
+    applyMiddleware(thunk, uiMiddleware),
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
can we pull this into another PR? no need to rush, but before merging it would be helpful to have it separated

Reply:
Yeah, for sure.